src/index.tsx

```diff
@@ -42,13 +50,22 @@ const useGoogleSheets = ({
         type: ActionTypes.success,
         payload: mappedData,
       });
+      dispatch({ type: ActionTypes.called, payload: true });
```

Comment:
No need to add the dispatch for `called` here and in catch because it will be called in finally block

Reply:
Good catch ! I didn't see those ones. It's all fixed now.